src/component/addedittimeline/index.tsx

```diff
@@ -115,8 +113,8 @@ function AddEditTimeline(props: AddEditTimelineProps) {
           membersWrite: data.membersWrite,
           ventureId,
           visibility,
-          successUrl: `../${data.timelineName}/feed`,
           token,
+          redirectOnSuccess: true,
```

Comment:
We don't know the created ID until the API response, so we just indicate if we want to redirect and let the hook take care of the final path.